app/utils/ethConstants.js

```diff
@@ -419,3 +419,36 @@ export const rewardsQuery = `
     }
   }
 `;
+
+const history = `
```

Comment:
This query returns too many fields. Only fields that are actually used on the UI must be included.